alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/executors/TikaJavaExecutor.java

```diff
@@ -109,9 +118,14 @@ public void transform(String transformName, String sourceMimetype, String target
         final boolean includeContents = parseBoolean(
                 transformOptions.getOrDefault(RequestParamMap.INCLUDE_CONTENTS, "false"));
         final boolean notExtractBookmarksText = parseBoolean(
-                transformOptions.getOrDefault(NOT_EXTRACT_BOOKMARK_TEXT, "false"));
+                transformOptions.getOrDefault(NOT_EXTRACT_BOOKMARK_TEXT, String.valueOf(notExtractBookmarksTextDefault)));
```

Comment:
It might be out-of-scope but I wonder if we should consider renaming "NOT_EXTRACT_BOOKMARK_TEXT" to "NOT_EXTRACT_BOOKMARKS_TEXT" ?

https://github.com/Alfresco/alfresco-transform-core/blob/master/alfresco-transformer-base/src/main/java/org/alfresco/transformer/util/RequestParamMap.java#L73

This would then also be consistent with:

https://github.com/Alfresco/alfresco-transform-core/blob/master/alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/executors/Tika.java#L464

`public static final String NOT_EXTRACT_BOOKMARKS_TEXT = "--notExtractBookmarksText";`